packages/react-components/src/components/VideoGallery/RemoteContentShareReactionOverlay.tsx

```diff
@@ -0,0 +1,244 @@
+        if (participant?.reaction) {
+          updateVisibleReactions(participant.reaction, participant.userId);
+        }
+      });
+    }, [remoteParticipantReactions, remoteParticipants, updateVisibleReactions]);
+
+    // Note: canRenderReaction shouldn't be needed as we remove the animation on the onAnimationEnd event
+    const canRenderReaction = (reaction: Reaction, id: string, movementIndex: number): boolean => {
+      // compare current time to reaction.received at and see if more than 4 seconds has elapsed
+      const canRender = Date.now() - getReceivedUnixTime(reaction.receivedOn) < REACTION_SCREEN_SHARE_ANIMATION_TIME_MS;
+
+      // Clean up the reaction if it's not in the visible reaction list
+      if (!canRender) {
+        removeVisibleReaction(reaction?.reactionType, id, movementIndex);
+      }
+
+      return canRender;
+    };
+
+    const containerHeight = hostDivHeight ?? 0;
+    const containerWidth = hostDivWidth ?? REACTION_START_DISPLAY_SIZE - REACTION_START_DISPLAY_SIZE;
+
+    const styleBucket = (activeSprites: number): IReactionStyleBucket => getReactionStyleBucket(activeSprites);
+    const displaySizePx = (activeSprites: number): number =>
+      REACTION_START_DISPLAY_SIZE * styleBucket(activeSprites).sizeScale;
+
+    const leftPosition = (position: number): number =>
+      generateStartPositionWave(position, containerWidth / 2, true /* isOriginAtCanvasCenter */);
+    const reactionMovementStyle = (position: number): React.CSSProperties =>
+      getReactionMovementStyle(leftPosition(position));
+
+    return (
+      <Stack
+        className={mergeStyles(videoContainerStyles, {
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          backgroundColor: 'transparent'
+        })}
+      >
+        {visibleReactions.map((reaction) => (
+          <div key={reaction.id} style={reactionOverlayStyle}>
+            <div className="reaction-item">
+              {canRenderReaction(reaction.reaction, reaction.id, reaction.reactionMovementIndex) && (
+                // First div - Section that fixes the travel height and applies the movement animation
+                // Second div - Keeps track of active sprites and responsible for marking, counting and removing reactions
+                // Third div - Responsible for opacity controls as the sprite emoji animates
+                // Fourth div - Responsible for calculating the point of X axis where the reaction will start animation
+                // Fifth div - Play Animation as the other animation applies on the base play animation for the sprite
+                <div
+                  style={moveAnimationStyles(
+                    containerHeight / 2, // dividing by two because reactionOverlayStyle height is set to 50%
+                    (containerHeight / 2) * (1 - reaction.styleBucket.heightMaxScale)
+                  )}
+                >
+                  <div>
```

Comment:
now this is empty we should be able to remove it (then update the div comments - those are excellent comments btw - super helpful!!)